### DISCUSSION TITLE
Having a method named search() does not work in v3

### DISCUSSION THREAD

**Discussion by timgavin** [livewire/livewire#6945] [7. Bugs (archived)] [answered]
### Livewire version

3.0.5

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

I built a search form in Livewire 2 and named my method `search()`. This worked, but after upgrading to 3 it breaks; the `search()` method will not be accessed.

```html
<div>
    <form wire:submit="search">
        <label for="search">
            <input type="search" wire:model="search" class="p-2 input md:p-3">
        </label>
        <button type="submit" class="mt-2 w-full btn">
            search
        </button>
    </form>
</div>
```

```php
public function search()
{
    dd('search');
}
```

Renaming `search` to `save` results in a working component.

```html
<div>
    <form wire:submit="save">
        <label for="search">
            <input type="search" wire:model="search" class="p-2 input md:p-3">
        </label>
        <button type="submit" class="mt-2 w-full btn">
            search
        </button>
    </form>
</div>
```

```php
public function save()
{
    dd('save');
}
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment, marked as answer]
Hi there!
In Livewire v3 you can't use the same method and property name as mentioned in the [upgrade docs](https://livewire.laravel.com/docs/upgrading#duplicate-method-and-property-names).

**Comment by timgavin** [comment]
@PhiloNL  hahaha the example even uses `search`. 🤦🏻‍♂️ Thanks, Philo!
